src/modules/dashboard/components/CoinMachine/CoinMachine.tsx

```diff
@@ -195,6 +196,7 @@ const CoinMachine = ({
   return (
     <div className={styles.main}>
       <BreadCrumb elements={breadCrumbs} />
+      {!transactionHash && <Confetti colonyAddress={colonyAddress} />}
```

Comment:
I'm not sure if this needs to check if transaction hash exists or not since the `Confetti` component has it's own logic for displaying.

Reply:
We need, because if we not, it will show also on pages with SaleState component, and we dont want it there